actor-server/actor-core/src/main/protobuf/dialog.proto

```diff
@@ -239,6 +239,16 @@ message DialogCommands {
 
     message WriteMessageSelfAck {}
 
+    message QuotedMessage {
```

Comment:
do u mean I should just send apiMessageOutReference from MessagingHandlers to DialogExtension and then to DialogCommandHandlers?
thenDialogCommandHandlers build the body of quotedMessage?

I should add two parameter to SendMessageProto for holdin apiMessageOutReference. (to send them to DialogCommandHandlers)